dir.props

```diff
@@ -84,6 +84,28 @@
   <PropertyGroup>
     <EnableDotnetAnalyzers Condition="'$(EnableDotnetAnalyzers)'==''">true</EnableDotnetAnalyzers>
   </PropertyGroup>
+  <!-- temporary change until we merge dev/api to all the branches. packaging.targets defines OutOfBoxFramework items
+       which is going to be used by EnsureOOBFramework target. We need uap10.1 to be enforced as a separate target 
+       in NuGet packages. And to do that we are going to leverage EnsureOOBFramework. However, this target will fail
+       if it can't cind any compatible targets within the project.json. There are some libraries that doesn't support
+       uap10.1 compatible frameworks so defining uap10.1 within packaging.targets will break the build unless those
+       projects are annotated as not supporting uap10.1. Since uap10.1 changes are not on all corefx branches, this change
+       will cause build breaks. So Instead of defining it in packaging.targets (in buildtools), we're defining it on 
+       the branch that can handle uap10.1. Once uap10.1 is merged into the other branches we should move this OutOfBoxFramework
+       item definition into packaging.targets. -->
```

Comment:
Sure. It doesn't matter much which way we do it. I can change it the way you propose. 

I don't know what your plans are. But I believe there's a major packaging debt in order to have the proper resolution of UAP10.1 from the generations (NetStandard). Is there an issue tracking that?